packages/transport/src/modules/secrets/SecretController.ts

```diff
@@ -74,15 +73,15 @@ export class SecretController extends TransportController {
         description = "",
         validTo?: CoreDate
     ): Promise<SecretContainerCipher> {
-        const plainString: string = secret.serialize();
-        const plainBuffer: CoreBuffer = CoreBuffer.fromUtf8(plainString);
+        const plainString = secret.serialize();
+        const plainBuffer = CoreBuffer.fromUtf8(plainString);
 
-        const nonce: number = await this.increaseNonce();
-        const encryptionKey: CryptoSecretKey = await CoreCrypto.deriveKeyFromBase(await this.getBaseKey(), nonce, SecretController.secretContext);
+        const nonce = await this.increaseNonce();
+        const encryptionKey = await CoreCrypto.deriveKeyFromBase(await this.getBaseKey(), nonce, SecretController.secretContext);
 
-        const cipher: CryptoCipher = await CoreCrypto.encrypt(plainBuffer, encryptionKey);
-        const createdAt: CoreDate = CoreDate.utc();
-        const secretContainerInterface: ISecretContainerCipher = {
```

Comment:
It may not be a problem, but when I hover the cursor over the `secretContainerInterface` variable, I no longer see that it is an `ISecretContainerCipher`.

Reply:
good catch!